CHANGELOG.md

```diff
@@ -3,6 +3,93 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## 7.16.1
+
+_Sep 12, 2024_
+
+We'd like to offer a big thanks to the 12 contributors who made this release possible. Here are some highlights ✨:
+
+- 📊 Charts performance improvement
+- 🧑‍💻 New Data Grid [custom columns demo](https://mui.com/x/react-data-grid/custom-columns/#full-example)
```

Comment:
we never used this emoji before ... shouldn't we stick to the ones we use frequently?
Also what should this demonstrate? It's not clear to me.

Reply:
I think it demonstrates something like an user accessing the new demo, but I think @cherniavskii will know better what he meant

Reply:
We never really defined what the emoji meant though :laughing:

Reply:
Yeah, I didn't think much about the emoji choice, I just thought it fits here 😅